recipe/meta.yaml

```
@@ -35,6 +35,8 @@ test:
     - mesonpy
   commands:
     - pip check
+    # CPPFLAGS exports -DNDEBUG which is unwanted
+    - export CPPFLAGS=${CPPFLAGS]//-DNDEBUG/}
```

looks like a stray character snuck in
```suggestion
    - export CPPFLAGS=${CPPFLAGS//-DNDEBUG/}
```